content/en/lotus/manage/slasher-and-disputer.md

```diff
@@ -0,0 +1,84 @@
+
+The Lotus Slasher functionality is activated by simply providing additional flags to the `lotus daemon` launch command as shown below.
+
+```shell
+lotus daemon --slash-consensus=true --slashdb-dir="/your/path"
+```
+
+The `--slash-consensus` and `--slashdb-dir` flags are both required to succesfully invoke the Slasher service.
+
+The `--slasher-sender` flag is also provided to optionally set the account to report consensus faults from.
+
+
+# The Lotus Disputer
+
+Prior to the introduction of [FIP-0010](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0010.md), Storage Providers were required to verify and record proofs of continued data storage in the form of on-chain WindowPoST messages. This approach often led to high costs and bandwidth usage associated with verifying proofs, especially during periods of congestion and rising gas fees. 
+
+To address this issue, [FIP-0010](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0010.md) was proposed and accepted to optimistically accept and record the proofs off-chain, reducing the burden on Storage Providers and network bandwidth by eliminating the need for extensive on-chain proof checking.
+
+Third-party Lotus node operators now enable off-chain verification of accepted WindowPoST proofs through the use of the Lotus Disputer service, utilizing the `DisputeWindowedPoSt` functionality.
+
+In the event that a Lotus Disputer successfully challenges an optimistically accepted Window PoST, the Storage Provider responsible for the faulty proof is fined based on the proportion of the expected block reward they would have received from each incorrectly proved sector. Additionally, all sectors with incorrect proofs are appropriately labelled as faulty. 
+
+The successful Lotus Disputer operator is also rewarded with a fixed `DisputeReward`, which is presently set at 4FIL.
+
+Any Lotus node operator is eligible to run a Lotus Disputer as outlined below.
+
+
+## Configure the Disputer
+
+The Lotus Disputer is already packagaed with Lotus so no additional installation steps are required. You **do not** need to be running `lotus-miner` to operate a Disputer.
+
+Once the node is fully synced, you can configure `lotus chain disputer` with the following options:
+
+```shell
+   --max-fee value  Spend up to X FIL per DisputeWindowedPoSt message
+   --from value     optionally specify the account to send messages from
+```
+
+## Run the Disputer
+
+Simply run `lotus chain disputer start` to start the disputer. The Disputer will automatically begin to dispute proofs from the current epoch unless a specific height is set using the `--start-epoch` flag.
+
+{{< alert icon="tip" >}}
+The Lotsu Disputer will take a couple of minutes to fully start. Any `rpc output message buffer` warning messages displayed during the the initiation process can be safely ignored.
+{{< /alert >}}
+
+## Manual Disputes
```

Comment:
There's nothing incorrect here, but I'd propose dropping this section -- there's no real situation when a user will want to do this.

Honestly, we might want to hide this command entirely, it only really exists for manual testing.

Reply:
Agreed!! I can't think of any situation where this would be used!